message_sender/factory.py

```diff
@@ -59,9 +79,60 @@ def _raw_send(self, py_data):
         res = r.json()
         return res.get('result', {})
 
+    def _override_payload(self, payload):
+        if self.override_payload:
+            old_payload = payload
+
+            def get_value(data, key):
+                index = key.find('.')
+                if index != -1:
+                    nested_data = data.get(key[:index], {})
+                    nested_key = key[index+1:]
+
+                    return get_value(nested_data, nested_key)
```

Comment:
I think a more pythonic way of doing this would be:
```
try:
    parent_key, nested_key = key.split('.', 1)
    return get_value(data.get(parent_key, {}), nested_key)
except ValueError:
    return data.get(key, key)
```